styles/main.css

```diff
@@ -132,8 +148,16 @@ footer hr{
 }
 
 .back-to-top{
-	    margin-right: -.5em;
+	margin-right: -.5em;
+    text-align: right;
     opacity: .5;
-	float:right;
```

Comment:
Use `text-align: right` instead of `float: right` here so that the nested hyperlink is in the correct horizontal position even when we set its position to fixed